crates/vault-registry/src/lib.rs

```diff
@@ -1155,6 +1170,7 @@ impl<T: Config> Pallet<T> {
 
         let to_slash = vault.liquidate(status)?;
         ext::sla::event_update_vault_sla::<T>(&vault.id(), ext::sla::Action::Liquidate)?;
+        ext::staking::force_refund::<T>(&vault_id)?;
```

Comment:
Not necessary, liquidations will still work without force refunds.

Reply:
It was a requirement in the design doc that nominations are refunded upon liquidation and banning - we can reconsider these

Reply:
They are refunded after liquidation, just as the Vault is - they can withdraw any leftover collateral whenever.